lib/util/platform.js

```diff
@@ -562,6 +562,10 @@ shaka.util.Platform = class {
         Platform.isWebOS4() || Platform.isWebOS5()) {
       return false;
     }
+    // Older chromecasts without GoogleTV seem to not support SMOOTH properly.
+    if (Platform.isChromecast() && !Platform.isAndroid()) {
```

Comment:
Can you add a new Platform? Something like:

  ```
/**
   * Check if the current platform is a Android-based Cast devices.
   *
   * @return {boolean}
   */
  static isAndroidCastDevice() {
    return shaka.util.Platform.isChromecast() &&
        shaka.util.Platform.userAgentContains_('Android');
  }
```

Reply:
Done!